2023/assignments/fiesty-cushion/fiesty-cushion.md

```diff
@@ -0,0 +1,135 @@
+# <p align='center'>Git Workshop 2023 <p>
+
+### <p align='center'>Braj Neupane</p>
```

Comment:
@Fiesty-Cushion, is this your authentic name? If not, kindly replace it, it will be used in your certificate.

Reply:
Yes lol